1.11/release-notes/1.11.0-beta3/index.md

```diff
@@ -0,0 +1,86 @@
+- Pods now support persistent volumes. [View the documentation](/1.11/deploying-services/pods).
+[/enterprise]
+
+## Metrics
+- Metrics - The DC/OS metrics component now produces metrics in Prometheus format. [View the documentation](/1.11/metrics).
+
+## Platform
+- Fault domain awareness. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed. [View the documentation](/1.11/deploying-services/fault-domain-awareness). [enterprise type="inline" size="small" /]
+- Linked clusters. A cluster link is a _**unidirectional**_ relationship between a cluster and another cluster. You add and remove links from one cluster to another cluster using DC/OS CLI. Once a link is set up you can easily switch between clusters using the CLI or UI. [View the documentation](/1.11/administering-clusters/multiple-clusters/cluster-links). [enterprise type="inline" size="small" /]
+- UCR
+  - Support for Docker image garbage collection. [View the documentation](/1.11/deploying-services/containerizers).
+  - Support for Docker image pull secrets.
+
+## Updated DC/OS data services
+- TLS auth for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS is now supported.
+- Fault domain awareness for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic and DC/OS HDFS. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed.
+- New API endpoint to pause a node for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. Use this endpointd to relaunch a node in an idle command state for debugging purposes.
+- New beta DC/OS Kafka ZooKeeper service. [View the documentation](/services/beta-zookeeper/index.md).
+- You can now select a DC/OS data service version from a dropdown menu in the DC/UI.
+- Improved scalability for all DC/OS data services.
+- Non-essential tasks now supported for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. If non-essential tasks fail, the entire node will not fail.
+
+# Breaking changes
+
+- Upgrades from 1.10 to 1.11 are _not supported_ in 1.11 Beta 3.
+
+# <a name="known-issues"></a>Known Issues and Limitations
+
+
+# <a name="fixed-issues"></a>Major Issues Fixed in 1.11 Beta 3
+
+- DCOS-16368 -  DC/OS Enterprise Secret Store no longer accepts secrets with uppercase letters in their paths. Secret paths with uppercase letters conflicted with use of the IAM API. [enterprise type="inline" size="small" /]
+- DCOS-19573 - Bugix: Changes to unique constraints now supported in the UI.
+- DCOS-19837 - Consolidate fault-domain scripts for all cloud providers into one script to support clusters with multiple cloud providers.
+- DCOS-19896 - `--linked` flag added to `dcos cluster list` so users can see which clusters can be unlinked. [enterprise type="inline" size="small" /]
```

Comment:
Some of these sound like improvements, but the heading says "major issues fixed". Intentional?